### DISCUSSION TITLE
"I am currently in 10th grade and recently applied for the Student Developer Pack, submitting my school ID as proof. I created my GitHub account just two days ago. However, my application was unfortunately rejected. I'm seeking guidance on how to proceed further."

### DISCUSSION THREAD

**Discussion by Umanmjha** [community/community#57509] [GitHub Education]
### Hi everyone,

"I am currently in 10th grade and recently applied for the Student Developer Pack, submitting my school ID as proof. I created my GitHub account just two days ago. However, my application was unfortunately rejected. I'm seeking guidance on how to proceed further."

**Comment by mrepol742** [comment]
Did you use your school issued email address? If not you should since its mandatory to use school issued email address.

**Comment by Umanmjha** [comment]
no my school not issue any email id then what should i do ?

**Comment by samus-aran** [comment]
Hi @Umanmjha  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
